features/onbekend_waardes.feature

```diff
@@ -3,87 +3,298 @@
-  - verblijfplaats.indicatieVestigingVanuitBuitenland krijgt de waarde true wanneer verblijfplaats.datumVestigingInNederland een waarde heeft (incl. 00000000)
-  - verblijfplaats.vanuitVertrokkenOnbekendWaarheen krijgt de waarde true wanneer verblijfplaats.landVanwaarIngeschreven = 0000
-  - verblijfplaats.verblijfBuitenland.vertrokkenOnbekendWaarheen krijgt de waarde true wanneer verblijfplaats.verblijfBuitenland.land = 0000.
-  - overlijden.indicatieOverleden: zie feature onvolledige_datum.feature
-
-
-  Scenario: een onbekendwaarde wordt niet opgenomen in de response
-    Gegeven in de registratie heeft de straatnaam van de verblijfplaats de waarde "."
-    Als de persoon wordt geraadpleegd
-    Dan is in het antwoord verblijfplaats.straatnaam niet aanwezig
+  Wanneer in de registratie specifieke waarden gereserveerd zijn voor een onbekende waarde, worden deze waarden niet doorgegeven in de API.
+  Wanneer een element in de registratie een standaardwaarde heeft, die betekent dat de waarde onbekend is, wordt het corresponderende veld niet opgenomen in de response.
+
+
+Rule: een veld wordt niet opgenomen wanneer het de standaardwaarde bevat
+  Het gaat om de volgende properties en standaardwaardes van de persoon:
+  | property                                         | standaardwaarde  |
+  | ------------------------------------------------ | ---------------- |
+  | naam.geslachtsnaam                               | .                |
+  | verblijfplaats.straat                            | .                |
+  | verblijfplaats.huisnummer                        | 0                |
+  | verblijfplaats.woonplaats                        | .                |
+  | verblijfplaats.nummeraanduidingIdentificatie     | 0000000000000000 |
+  | verblijfplaats.adresseerbaarObjectIdentificatie  | 0000000000000000 |
+  | ouders.naam.geslachtsnaam                        | .                |
+  | partners.naam.geslachtsnaam                      | .                |
+  | partners.soortVerbintenis                        | .                |
+  | kinderen.naam.geslachtsnaam                      | .                |
+  | reisdocumentnummers                              | .........        |
+
+  Abstract Scenario: onbekend waarde "<waarde>" voor <veld>
+    Gegeven het systeem heeft een persoon met de volgende gegevens
+    | naam                | waarde     |
+    | burgerservicenummer | 999992934  |
+    En de persoon heeft de volgende <groep> gegevens
+    | naam   | waarde   |
+    | <veld> | <waarde> |
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                             |
+    | type                | RaadpleegMetBurgerservicenummer    |
+    | burgerservicenummer | 999992934                          |
+    | fields              | burgerservicenummer,<groep>.<veld> |
+    Dan bevat de persoon met burgerservicenummer '999992934' alleen de volgende gegevens
```

Comment:
Kan deze 'dan' stap als volgt worden herschreven:

```
    Dan bevat de persoon met burgerservicenummer '999992934' de volgende '<groep>' gegevens NIET
    | naam   |
    | <veld> |
```

Deze stap is al geïmplementeerd en wordt in o.a. de woonplaats feature gebruikt om te valideren dat een VerblijfplaatsBuitenland geen woonplaats property heeft

Reply:
done, maar daarmee check je dat <veld> niet voorkomt in <groep>, terwijl hierdoor <groep> zelf ook niet voorkomt (we nemen geen leeg object op)

Reply:
Deze situatie treedt alleen op voor de geslachtsnaam scenario. Bij verblijfplaatsen komt altijd de type property terug. Ik denk dat het in dit geval het makkelijkste is om de automation code hiermee rekening te laten houden ipv deze abstract scenario te splitsen in twee abstract scenario's